libs/functions/generate_api.py

```diff
@@ -83,75 +85,68 @@ def _generate_api_for_projections(projection_row):
     )
     return projections
 
-
-def _generate_state_actuals(
-    projection_row: pd.Series, state_intervention: Intervention, state_beds_data: dict
-):
-    """Generates Actuals for a state.
-
-    Args:
-        projection_row: Output from projection DataFrame.
-        state_intervention: Intervention for state
-        state_beds_data: Bed data for a specific state.
-    """
-    intervention_str = state_intervention.name
-
-    return _Actuals(
-        population=projection_row[rc.POPULATION],
-        intervention=intervention_str,
-        cumulativeConfirmedCases=projection_row[rc.CURRENT_CONFIRMED],
-        cumulativeDeaths=projection_row[rc.CURRENT_DEATHS],
-        cumulativePositiveTests=_get_or_none(
-            projection_row[rc.CUMULATIVE_POSITIVE_TESTS]
-        ),
-        cumulativeNegativeTests=_get_or_none(
-            projection_row[rc.CUMULATIVE_NEGATIVE_TESTS]
-        ),
-        hospitalBeds={
-            "capacity": projection_row[rc.PEAK_BED_CAPACITY],
+def _generate_actuals(actual_data, intervention_str): 
+    hospital_beds = None
+    if CommonFields.MAX_BED_COUNT in actual_data and _get_or_none(actual_data[CommonFields.MAX_BED_COUNT]):
+        hospital_beds = {
+            "capacity": actual_data[CommonFields.MAX_BED_COUNT],
```

Comment:
Can you make capacity an optional field? then we can have this field without any if statements.

Also, you should be able to get `currentUsage` from `actual_data[CommonFields.CURRENT_HOSPITALIZED]` (or something like that).